openedx/core/djangoapps/profile_images/exceptions.py

```diff
@@ -0,0 +1,18 @@
+"""
+Exceptions related to the handling of profile images
+"""
+
+from django.utils.translation import ugettext as _
+
+
+class ImageValidationError(Exception):
+    """
+    Exception to use when the system rejects a user-supplied source image.
+    """
+    @property
+    def user_message(self):
+        """
+        Translate the developer-facing exception message for API clients.
+        """
+        # pylint: disable=translation-of-non-string
+        return _(self.message)
```

Comment:
To ensure translation can happen, make sure that all possible values of `self.message` are flagged for translation with `uggettext_noop` when they are defined.

https://docs.djangoproject.com/en/1.9/ref/utils/#django.utils.translation.ugettext_noop
https://www.gnu.org/software/gettext/manual/html_node/Special-cases.html